lib/components/app/app-menu.js

```diff
@@ -41,9 +47,93 @@ class AppMenu extends Component {
     window.location.href = startOverUrl
   }
 
+  _expandPane = () => {
+    const { isPaneOpen } = this.state
+    this.setState({ isPaneOpen: !isPaneOpen })
+  }
+
+  _expandSubmenu = (id) => {
+    const { expandedSubmenus } = this.state
+    const currentlyOpen = expandedSubmenus[id] || false
+    this.setState({ expandedSubmenus: { [id]: !currentlyOpen } })
+  }
+
+  _addExtraMenuItems = (menuItems) => {
+    const AddIconAndLabel = ({iconType, iconUrl, label}) => {
+      return (
+        iconUrl
+          ? <span>
+            <img alt={`Icon for ${label} menu item`} src={iconUrl} />
```

Comment:
Now that localization is well under way and that this PR depends on #435, I'm going to recommend to extract this string to the i18n YML files.

Reply:
Prep added in 542d756098ceae9ad56f74334fae242b4be7c938